### ISSUE TITLE
Tizen support

### ISSUE THREAD

**Issue by F1nZeR** [xamarin/Essentials#23]
Are you interested in Tizen support?
As I can see more and more xam-plugins now supports tizen platform.

**Comment by Redth** [comment]
It's definitely something we are considering in the future, but currently our focus for the first release is iOS, Android, and UWP.

**Comment by rookiejava** [comment]
well.. I promsie I'll contribute the tizen backend soon. :)

**Comment by mattleibow** [comment]
@rookiejava I think we are good enough that we can start taking PRs for Tizen. I have created a new branch [`feature/tizen-support`](https://github.com/xamarin/Essentials/tree/feature/tizen-support) and I also have a PR that you can see what things look like so far: #244 

Go wild!

**Comment by mattleibow** [comment]
I think it is safe to start adding features for Tizen. I started a branch, [`feature/tizen-support`](https://github.com/xamarin/Essentials/tree/feature/tizen-support) where PRs should go against.

**Comment by Redth** [comment]
Indeed, the Tizen team is on this.  They have done some work and will need to refactor against our API breaks which will be merged soon.  We will figure out how we incorporate this going forward.
